packages/generator/test/generator.spec.ts

```diff
@@ -1,12 +1,36 @@
 /* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+import * as path from 'path';
 import { ODataVersion } from '@sap-cloud-sdk/util';
 import { SourceFile, ClassDeclaration, FunctionDeclaration } from 'ts-morph';
-import { generateProject } from '../src';
+import { generateProject, GeneratorOptions } from '../src';
 import { createOptions } from './test-util/create-generator-options';
 
 describe('generator', () => {
   let files: SourceFile[];
 
+  const pathToTestFile = path.resolve(
+    process.cwd(),
+    '../../test-resources/some-test-markdown.md'
+  );
+  it('copies the additional files matching the glob.', async () => {
+    const project = await generateProject(
+      createOptions({
+        inputDir: '../../test-resources/service-specs/v2/API_TEST_SRV',
+        additionalFiles: '../../test-resources/*.md'
+      })
+    );
+
+    expect(
+      project!
+        .getSourceFiles()
+        .filter(
+          file =>
+            file.getBaseName() === 'some-test-markdown.md' ||
+            file.getBaseName() === 'CHANGELOG.md'
+        ).length
+    ).toBe(2);
```

Comment:
This seems like a weird construction, maybe:

```suggestion
    const files = project!.getSourceFiles();
    expect(files.find(file => file.getBaseName() === 'some-test-markdown.md')).toBeDefined();
    expect(files.find(file => file.getBaseName() === 'CHANGELOG.md')).toBeDefined();
```